### ISSUE TITLE
ConnectWallet does nothing in Safari, opens Phantom in Chrome, and nav's to Brave wallet in Brave 🤔

### ISSUE THREAD

**Issue by richardrauser** [coinbase/onchainkit#723]
### Describe the bug and the steps to reproduce it

I use many different wallets in my desktop browsers and am finding the OnchainKit ConnectWallet button a little troublesome. Here is an outline of the behaviour:

**Safari:** tapping the ConnectWallet button does nothing. Obviously I don't have any wallets installed in desktop Safari, but doing nothing doesn't seem like a good way to handle this.

**Chrome:** I have MetaMask, Coinbase Wallet, and Phantom installed here, amongst other wallets. Tapping the ConnectWallet opens Phantom. No option to select MM or CBW instead is provided, which would be incredibly helpful.

**Brave:** I have just MetaMask installed here. Instead of connecting to MetaMask, I am routed to Brave's wallet page: brave://wallet/crypto/onboarding/welcome

The docs don't explain how I might modify the behaviour of ConnectWallet to address this.

At present this makes the ConnectWallet unusable for me. Would love to see these issues addressed.

### What's the expected behavior?

Users should be able to select the wallet they'd like to use.

### What version of the libraries are you using?

-     "@coinbase/onchainkit": "^0.23.2"

**Comment by kyhyco** [comment]
Ah we haven't tested with multiple wallets loaded into the browsers. This is a great call out!

**Comment by kyhyco** [comment]
Yes you can. We use wagmi under the hood. So you should be able to use any other wallet provider that uses wagmi. I think walletconnect or privy should work.

**Comment by kyhyco** [comment]
We are releasing a new Wallet component that will replace `ConnectAccount`. FYI, think it will have the same issue that you experienced here. We will take a look at this shortly.

**Comment by kyhyco** [comment]
Ok actually check your wagmi config. This is what we have on the doc site:

```typescript
import { WagmiProvider, createConfig, http } from 'wagmi';
import { base } from 'wagmi/chains';
import { coinbaseWallet } from 'wagmi/connectors';

const wagmiConfig = createConfig({
  chains: [base],
  connectors: [
    coinbaseWallet({
      appName: 'onchainkit',
    }),
  ],
  ssr: true,
  transports: {
    [base.id]: http(),
  },
});
```

Haven't tested multi wallet browser plugin but this should pop up the smart wallet _if_ user does not have any wallet browser extensions installed.

**Comment by richardrauser** [comment]
@kyhyco thanks for the tips! I've managed to get this all working with some finessing of wagmi config.. you can see an example of this on a project I'll be submitting to the Base Onchain Summer hackathon this week!  

https://www.anglez.xyz/

Still needs a little work 😁
